### PR TITLE
Update Hammurabi example code to v3

### PR DESCRIPTION
**Pull request by thomashacker**
## Description

The current example code of the universe pipeline [Hammurabi](https://spacy.io/universe/project/hmrb) does not work. The updated v3 code is from the [Hammurabi repository](https://github.com/babylonhealth/hmrb/blob/master/examples/spacy3.py).

> Unfortunately, due to npm issues, I couldn't get the website preview working and verify if the changes cause any problems.

### Types of change

Change to documentation

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -3238,34 +3238,62 @@
             "github": "babylonhealth/hmrb",
             "pip": "hmrb",
             "code_example": [
-                "import spacy # __version__ 3.0+",
+                "import spacy",
                 "from hmrb.core import SpacyCore",
```
Comment (svlandeg) [inline website/meta/universe.json:3242]:
```suggestion
```
Looks like this import can be removed?

File: website/meta/universe.json [hunk 2]
```diff
@@ -3238,34 +3238,62 @@
...
+                "   },",
                 "   \"map_doc\": \"augmenters.jsonify_span\",",
                 "   \"sort_length\": True,",
                 "}",
-                "nlp = spacy.load(\"en_core_web_sm\")",
-                "nlp.add_pipe(\"hammurabi\", config=conf)",
-                "nlp(sentences)"
+                "",
+                "nlp.add_pipe(\"hmrb\", config=conf)",
+                "nlp(sentence)"
```
Comment (svlandeg) [inline website/meta/universe.json:3296]:
```suggestion
                "nlp(sentences)"
```

**Review by svlandeg — COMMENTED** [review]
Great to see this updated! 

The formatting looks correct to me - just had two small comments.

File: website/meta/universe.json (hunk 1 above)
Comment (thomashacker) [reply website/meta/universe.json:3242]: It's is important for the `nlp.add_pipe(\"hmrb\", config=conf)` to find the factory

**Pull request merged**